src/algebra/module/submodule/lattice.lean

```diff
@@ -249,6 +249,13 @@ lemma eq_zero_of_coe_mem_of_disjoint (hpq : disjoint p q) {a : p} (ha : (a : M)
   a = 0 :=
 by exact_mod_cast disjoint_def.mp hpq a (coe_mem a) ha
 
+/-- A submodule is maximal if it is maximal in the collection of proper submodules. -/
+class is_maximal (N : submodule R M) : Prop := (out : is_coatom N)
```

Comment:
Since this is linked from the `undergrad.yaml`, you should add something like
```suggestion
/-- A submodule is maximal if it is maximal in the collection of proper submodules.

Note that when `M = R`, this is the definition of a maximal ideal.
-/
class is_maximal (N : submodule R M) : Prop := (out : is_coatom N)
```